spec/enum.dd

```diff
@@ -123,8 +123,12 @@ enum E : C
 ---
 )
 
-        $(P An empty enum body (For example $(CODE enum E;)) signifies an opaque
-        enum - the enum members are unknown.)
+        $(P An empty enum body signifies an opaque enum - the enum members are unknown.)
+        ---
+        enum X;          // opaque enum
+        writeln(X.init); // error: enum x is opaque and has no default initializer
+                         // See 17.1.1 to understand what .init means
```

Comment:
There's no need for this line.

Reply:
I made a new commit where I erased that line. Also I added a proper message to the commit.